src/EditorFeatures/CSharpTest/Diagnostics/UseAutoProperty/UseAutoPropertyTests.cs

```diff
@@ -1000,5 +1000,34 @@ int Q
     int Q { get; }
 }", fixAllActionEquivalenceKey: FeaturesResources.Use_auto_property);
         }
+
+        [WorkItem(23735, "https://github.com/dotnet/roslyn/issues/23735")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseAutoProperty)]
+        public async Task ExplicitInterfaceImplementation()
```

Comment:
:bulb: It would be good to add a second test demonstrating the behavior for explicit interface properties that have both `get` and `set`.